plugin/cr/src/test/resources/ersd-small-active-bundle.json

```diff
@@ -1,449 +1,461 @@
-              },
-              "expansion": {
-                  "timestamp": "2022-10-21T15:18:29-04:00",
-                  "contains": [
-                      {
-                          "system": "http://snomed.info/sct",
-                          "version": "Provisional_2022-04-25",
-                          "code": "1086051000119107"
-                      },
-                      {
-                          "system": "http://snomed.info/sct",
-                          "version": "Provisional_2022-04-25",
-                          "code": "1086061000119109"
-                      },
-                      {
-                          "system": "http://snomed.info/sct",
-                          "version": "Provisional_2022-04-25",
-                          "code": "1086071000119103"
-                      }
-                  ]
-              }
-          },
-          "request": {
-              "method": "PUT",
-              "url": "ValueSet/123-this-will-be-routine"
-          }
-      }
-  ]
-}
\ No newline at end of file
+	"resourceType": "Bundle",
+	"id": "rctc-release-2022-10-19-Bundle-rctc",
+	"type": "transaction",
+	"timestamp": "2022-10-21T15:18:28.504-04:00",
+	"entry": [
+		{
+			"fullUrl": "http://ersd.aimsplatform.org/fhir/Library/SpecificationLibrary",
+			"resource": {
+				"resourceType": "Library",
+				"id": "SpecificationLibrary",
+				"url": "http://ersd.aimsplatform.org/fhir/Library/SpecificationLibrary",
+				"version": "2022-10-19",
+				"status": "active",
+				"relatedArtifact": [
+					{
+						"type": "composed-of",
+						"resource": "http://ersd.aimsplatform.org/fhir/PlanDefinition/us-ecr-specification|2.0.0",
+						"extension": [
+							{
+								"url": "http://hl7.org/fhir/StructureDefinition/crmi-isOwned",
+								"valueBoolean": true
+							}
+						]
+					},
+					{
+						"type": "composed-of",
+						"resource": "http://ersd.aimsplatform.org/fhir/Library/rctc|2022-10-19",
+						"extension": [
+							{
+								"url": "http://hl7.org/fhir/StructureDefinition/crmi-isOwned",
+								"valueBoolean": true
+							}
+						]
+					},
+					{
+						"type": "composed-of",
+						"resource": "http://notOwnedTest.com/Library/notOwnedRoot|0.1.1"
+					},
+					{
+						"extension": [
+							{
+								"url": "http://aphl.org/fhir/vsm/StructureDefinition/vsm-valueset-priority",
+								"valueCodeableConcept": {
+									"coding": [
+										{
+											"system": "http://hl7.org/fhir/us/ecr/CodeSystem/us-ph-usage-context",
+											"code": "emergent"
+										}
+									],
+									"text": "Emergent"
+								}
+							},
+							{
+								"url": "http://aphl.org/fhir/vsm/StructureDefinition/vsm-valueset-condition",
+								"valueCodeableConcept": {
+									"coding": [
+										{
+											"system": "http://snomed.info/sct",
+											"code": "49649001"
+										}
+									],
+									"text": "Infection caused by Acanthamoeba (disorder)"
+								}
+							}
```

Comment:
accidentally reformatted this file, the only meaningful change is here (55-65), the new extension was added